### ISSUE TITLE
[BUG] - edge wasn't removed

### ISSUE THREAD

**Issue by 1043613lcpsorg** [Atlas-OS/Atlas#1153]
### Before continuing...

- [X] I agree to follow Atlas' [Code of Conduct](https://github.com/Atlas-OS/.github/blob/main/profile/CODE_OF_CONDUCT.md)
- [X] I have searched our [issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to verify that the issue hasn't been reported already
- [X] I have searched the [Atlas documentation](https://docs.atlasos.net) beforehand for a solution
- [X] I am on the latest version of Atlas, and didn't do an unsupported upgrade

### Description

when is selected remove microsoft edge and install firefox microsoft edge stayed there

### Steps to reproduce

delete edge then edge stays there

### Expected behavior

edge was supposed to be removed

### Actual behavior

edge wasn't removed
![Annotation 2024-07-21 195505](https://github.com/user-attachments/assets/ebe5ff9c-2488-4e46-913e-fb04812876a4)

### Atlas Edition

Atlas for Windows 10 22H2

### Desktop information

OS Name	Microsoft Windows 10 Pro
Version	10.0.19045 Build 19045
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	DESKTOP-BPA141M
System Manufacturer	To Be Filled By O.E.M.
System Model	To Be Filled By O.E.M.
System Type	x64-based PC
System SKU	To Be Filled By O.E.M.
Processor	AMD Ryzen 5 1600 Six-Core Processor, 3200 Mhz, 6 Core(s), 12 Logical Processor(s)
BIOS Version/Date	American Megatrends Inc. P3.00, 7/13/2017
SMBIOS Version	3.0
Embedded Controller Version	255.255
BIOS Mode	UEFI
BaseBoard Manufacturer	ASRock
BaseBoard Product	AB350 Pro4
BaseBoard Version	
Platform Role	Desktop
Secure Boot State	Off
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\Windows
System Directory	C:\Windows\system32
Boot Device	\Device\HarddiskVolume3
Locale	United States
Hardware Abstraction Layer	Version = "10.0.19041.2728"
User Name	DESKTOP-BPA141M\ohios
Time Zone	Pacific Daylight Time
Installed Physical Memory (RAM)	32.0 GB
Total Physical Memory	31.9 GB
Available Physical Memory	29.5 GB
Total Virtual Memory	36.9 GB
Available Virtual Memory	34.5 GB
Page File Space	5.00 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	Off
Virtualization-based security	Not enabled
Device Encryption Support	Elevation Required to View
Hyper-V - VM Monitor Mode Extensions	Yes
Hyper-V - Second Level Address Translation Extensions	Yes
Hyper-V - Virtualization Enabled in Firmware	No
Hyper-V - Data Execution Protection	Yes

**Comment by Ast3risk-ops** [comment]
Hmm, if you're not on Atlas 0.4 then this is because you didn't update, for now just run this in Powershell to remove Edge:

```
iex(irm https://cdn.jsdelivr.net/gh/he3als/EdgeRemover@main/get.ps1)
```

**Comment by overwatch3560** [comment]
I have this too aswell as after install it isn't removed and also after using the program included in the atlas folder

**Comment by overwatch3560** [comment]
> Hmm, if you're not on Atlas 0.4 then this is because you didn't update, for now just run this in Powershell to remove Edge:
> 
> 
> 
> ```
> 
> iex(irm https://cdn.jsdelivr.net/gh/he3als/EdgeRemover@main/get.ps1)
> 
> ```

This solution actually works thank you

**Comment by he3als** [comment]
Fixed in [release v0.4.1](https://github.com/Atlas-OS/Atlas/releases/tag/0.4.1). See the release for more info on what v0.4 users can do.
